DataAccess/Entities/DatabaseContext.cs

```diff
@@ -17,12 +20,16 @@ public DatabaseContext(DbContextOptions<DatabaseContext> options) : base(options
     public DbSet<Star> Stars { get; set; }
     public DbSet<Tool> Tools { get; set; }
 
-    protected override void OnModelCreating(ModelBuilder modelBuilder)
-    {
-      modelBuilder.Entity<Member>().HasIndex(m => m.Email).IsUnique();
-      modelBuilder.Entity<Member>().HasIndex(m => m.Username).IsUnique();
-      SeedDatabase.SeedAll(modelBuilder);
-    }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Member>().HasIndex(m => m.Email).IsUnique();
+            modelBuilder.Entity<Member>().HasIndex(m => m.Username).IsUnique();
+            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development") ;
```

Comment:
syntax error here. remove the semi colon.
```suggestion
            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
```